src/vaadin-menu-bar.html

```diff
@@ -38,10 +38,19 @@
       [part="overflow-button"] {
         margin-right: 0;
       }
+
+      .dots::before {
+        display: block;
+        content: "\00B7\00B7\00B7";
```

Comment:
Wonder if we should use the horizontal ellipsis character (…) instead. @jouni ?

Reply:
Using horizontal ellipsis character (…) requires to do extra tweaks for vertical alignment, as I mentioned in the PR description above.

I was trying to align with the mockup from https://github.com/vaadin/vaadin-menu-bar/issues/8#issue-412905311

Reply:
Ah, missed that.